lib/eco/grammars/grammars.py

```diff
@@ -157,6 +180,11 @@ def __hash__(self):
 lang_dict = {}
 
 def add_lang(lang, new=False, sub=False):
+    for l in languages:
```

Comment:
Is this the same as `if lang in languages: return`?

Reply:
I think you're right!

Reply:
Fixed.